package/CHANGELOG

```diff
@@ -13,7 +13,15 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-mm/dd/18 orbeckst, PicoCentauri, richardjgowers
+mm/dd/yy
+  * 0.19.2
+
+Fixes
+
+Deprecations
+
+
+11/05/18 orbeckst, PicoCentauri, richardjgowers, zemanj
```

Comment:
Shouldn't this have something about Windows conda package? We can say experimental...

Reply:
Ah I guess... It's not even experimental tbh, all tests pass